astroid/mixins.py

```diff
@@ -28,6 +28,8 @@
 class BlockRangeMixIn:
```

Comment:
This mixin is not recognized as one, I'll fix this myself suggestions won't work here.